### PR TITLE
Chore/adding_infobutton

### PR DESCRIPTION
**Pull request by divyaocc**
I made necessary below changes for tooltip info button right next to each study filter category with additional data.
1. Updated the documentation at docs/portal_config.md  with new config.
2. Updated code in https://github.com/uc-cdis/data-portal/blob/master/src/Discovery/DiscoveryDropdownTagViewer.tsx and
https://github.com/uc-cdis/data-portal/blob/master/src/Discovery/DiscoveryTagViewer.tsx

### CODE REVIEW

File: src/Discovery/DiscoveryDropdownTagViewer.tsx [hunk 1]
```diff
@@ -113,16 +116,46 @@ const DiscoveryDropdownTagViewer: React.FunctionComponent<DiscoveryTagViewerProp
...
               xl={12}
               xxl={12}
+
             >
-              { tags }
+              <table>
+                <tr>
+                  <td>
+                  <Col>
+                  <div style={{ display: category.tooltip != null ? 'block' : 'none' }}>
```
Comment (mfshao) [inline src/Discovery/DiscoveryDropdownTagViewer.tsx:133]: don't use CSS to hide a mounted element. you should be doing something like this https://github.com/uc-cdis/data-portal/blob/master/src/Discovery/DiscoveryDetails.tsx#L234-L267 to not mounting an element in the first place

File: src/Discovery/DiscoveryDropdownTagViewer.tsx [hunk 2]
```diff
@@ -5,8 +5,11 @@ import {
 } from 'antd';
 import { DiscoveryConfig } from './DiscoveryConfig';
 import { DiscoveryResource } from './Discovery';
+import Tooltip from 'rc-tooltip';
+import { InfoCircleOutlined } from '@ant-design/icons'
 
 const { Option } = Select;
+//const tooltipText = ' test info';
```
Comment (mfshao) [inline src/Discovery/DiscoveryDropdownTagViewer.tsx:12]: is this needed?

File: src/Discovery/DiscoveryDropdownTagViewer.tsx [hunk 3]
```diff
@@ -113,16 +116,46 @@ const DiscoveryDropdownTagViewer: React.FunctionComponent<DiscoveryTagViewerProp
...
               xs={24}
               sm={24}
               md={12}
               lg={12}
               xl={12}
               xxl={12}
+
             >
-              { tags }
+              <table>
```
Comment (mfshao) [inline src/Discovery/DiscoveryDropdownTagViewer.tsx:129]: why we have to use a table instead something simpler like Grid or Space?

File: src/Discovery/DiscoveryTagViewer.tsx [hunk 4]
```diff
@@ -25,7 +27,7 @@ const DiscoveryTagViewer: React.FunctionComponent<DiscoveryTagViewerProps> = (pr
     const tagMap = {};
     studies.forEach((study) => {
       const tagField = props.config.minimalFieldMapping.tagsListFieldName;
-      study[tagField]?.forEach((tag) => {
+      study[tagField].forEach((tag) => {
```
Comment (mfshao) [inline src/Discovery/DiscoveryTagViewer.tsx:30]: why this change?

File: src/Discovery/DiscoveryTagViewer.tsx [hunk 5]
```diff
@@ -139,12 +141,42 @@ const DiscoveryTagViewer: React.FunctionComponent<DiscoveryTagViewerProps> = (pr
                 xl={6}
                 xxl={4}
               >
-                <h5 className='discovery-header__tag-group-header'>{categoryDisplayName}</h5>
-                { tags }
-              </Col>
-            );
-          })
-        }
+                <table>
```
Comment (mfshao) [inline src/Discovery/DiscoveryTagViewer.tsx:144]: all my comments above regarding to src/Discovery/DiscoveryDropdownTagViewer.tsx also applies to here

**Comment by ocshawn** [comment]
closing to cleanup prs if this is still an issue please address above cements and reopen
